### PR TITLE
feat: Azure Devops Pipeline support

### LINKED ISSUE #273: feat: Azure Devops Pipeline support
Add Azure Pipelines as to the list of supported runner types.

### PR DESCRIPTION
**Pull request by danlishka**
This has already been tested with local agent on Linux. I still want to spend more time validating it in the Azure cloud & k8s.

Documentation changes: https://github.com/chainloop-dev/docs/pull/143

Screenshot 2023-07-27 at 18 00 53
Screenshot 2023-07-27 at 17 55 05

closes #273

### CODE REVIEW

File: app/cli/internal/action/workflow_run_list.go [hunk 1]
```diff
@@ -115,6 +115,8 @@ func humanizedRunnerType(in v1.CraftingSchema_Runner_RunnerType) string {
 		return "GitHub"
 	case *v1.CraftingSchema_Runner_GITLAB_PIPELINE.Enum():
 		return "GitLab"
+	case *v1.CraftingSchema_Runner_AZURE_DEVOPS_PIPELINE.Enum():
+		return "Azure DevOps Pipeline"
```
Comment (migmartri) [inline app/cli/internal/action/workflow_run_list.go:119]: This information I believe will be shown in the tables, I am wondering if it's too long compared to the others.

In fact, [here](https://azure.microsoft.com/en-us/products/devops/pipelines#overview) they seem to reference to it as `Azure pipelines`, would it make sense to do the same here?

**Review by migmartri — APPROVED** [review]
LGTM! just a minor comment

File: app/cli/internal/action/workflow_run_list.go (hunk 1 above)
Comment (danlishka) [reply app/cli/internal/action/workflow_run_list.go:119]: > in fact, [here](https://azure.microsoft.com/en-us/products/devops/pipelines#overview) they seem to reference to it as Azure pipelines

Yes, I found both references and was already thinking about making it shorter. Thanks, I will do it.

Comment (danlishka) [reply app/cli/internal/action/workflow_run_list.go]: - Screenshot 2023-07-29 at 23 06 51
- Screenshot 2023-07-29 at 23 06 16

**Pull request merged**